dbms/src/Common/MyTime.cpp

```diff
@@ -683,6 +818,10 @@ std::pair<Field, bool> parseMyDateTimeAndJudgeIsDate(const String & str, int8_t
         }
         if (l == 5 || l == 6 || l == 8)
         {
+            if (isFloat)
```

Comment:
Better to add a TODO item to refact this code, to remove these isFloat branches.